src/components/Code.tsx

```diff
@@ -262,9 +262,9 @@ function renderLowLightNode(node: lowlight.AST.Unist.Node) {
 
 function renderLowLightTree(tree: lowlight.HastNode[]) {
   if (tree.length === 1) {
-    return renderLowLightNode(tree[0]);
+    return renderLowLightNode(tree[0], 0);
   } else {
-    return <>{tree.map(renderLowLightNode)}</>;
+    return <>{tree.map((node, index) => renderLowLightNode(node, index))}</>;
```

Comment:
>index [](start = 30, length = 5)

It looks like this is using the index of an item in a list as the key.  That isn't reliable the key must stay unique for the lifetime of the list item.  If you use the index, the diff'ing engine can be tricked into not producing a new item which would be a bug.  It can think item n is still the same even in the case that item n is a new item (either through an insert or a remove).  The guidance I've seen in the past is to use the text from the item as that is usually unique.  Note that I'm not an expert on generating unique keys, perhaps there's an internet article on best practices.  Here's an article that might help:  [https://stackoverflow.com/questions/39549424/how-to-create-unique-keys-for-react-elements](https://stackoverflow.com/questions/39549424/how-to-create-unique-keys-for-react-elements)

Reply:
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
There's some more info linked from the [React.js docs on keys](https://reactjs.org/docs/lists-and-keys.html#keys).

Reply:
Resolved.